Tuist/ProjectDescriptionHelpers/Module.swift

```diff
@@ -817,6 +810,15 @@ public enum Module: String, CaseIterable {
             releaseSettings["SWIFT_STRICT_CONCURRENCY"] = .string(strictConcurrencySetting)
         }
 
+        let rootFolder: String
+        switch product {
+        case .unitTests:
+            rootFolder = "Tests"
+            debugSettings["CODE_SIGN_IDENTITY"] = ""
```

Comment:
Fly-by fix – often, building the unit tests target fails due to signing. Since we don't need to sign those, this makes the builds more stable without any drawback.